src/styles/utils/variable.scss

```
@@ -0,0 +1,5 @@
+$color-main: #060b35;
+$color-secondary: #616070;
+$color-button-blue: #00acdc;
+$color-button-white: #fff;
```

You have variables but you don't use them. 
And also be careful and use more universal naming.
Example:
```suggestion
$color-button: #00acdc;
$color-button-active: #fff;
```